content/SCALE/SCALETutorials/Credentials/DirectoryServices/ConfigIDmapSCALE.md

```diff
@@ -1,23 +1,41 @@
 ---
 title: "Configuring IDMap"
-description: "This article provides instructions on configuring and managing IDMap in SCALE."
+description: "This article provides instructions on configuring and managing ID mapping in SCALE."
 weight: 35
 aliases:
 tags:
 - scaleidmap
-- scaledirectoryservices
 ---
 
 {{< toc >}}
 
+{{< hint danger >}}
+Only administrators experienced with configuring Id mapping should attempt to add new or edit existing idmaps. 
+Misconfiguration can impact system operation.
```

Comment:
Idmap settings exist for the purpose of integration with an existing active directory domain to ensure that UIDs and GIDs assigned to active directory users and groups have consistent values domain-wide. The correct configuration therefore relies on details that are entirely external to the truenas server, e.g. how the AD administrator has configured other Unix-like computers in the environment. 

The default is to use an algorithmic method of generating IDs based on the RID component of the user or group's SID in active directory.

Reply:
Misconfiguration may lead to permissions being incorrectly assigned to users or groups in the case where data is transferred to / from external servers via ZFS replication or rsync (or access is being performed via NFS or other protocols that directly access the UIDs / GIDs on files).